src/bundle.ts

```diff
@@ -15,12 +15,9 @@ import {Transform} from 'stream';
 import File = require('vinyl');
 import * as logging from 'plylog';
 import {urlFromPath} from './path-transformers';
-import {StreamAnalyzer, DepsIndex} from './analyzer';
-import {compose} from './streams';
+import {StreamAnalyzer} from './analyzer';
```

Comment:
very cool. I just finished up another PR to add this to tslint, I'll add it to tsconfig in that PR as well